content/pt-br/docs/concepts/overview/working-with-objects/kubernetes-objects.md

```diff
@@ -0,0 +1,90 @@
+ 
+
+
+
+
+### Descrevendo um objeto Kubernetes
+
+Quando se cria um objeto do Kubernetes, deve-se fornecer a especificação do objeto que descreve seu estado desejado, bem como algumas informações básicas sobre o objeto (como um nome, por exemplo). Quando utiliza a API Kubernetes para criar o objeto (diretamente ou via `kubectl`), essa solicitação de API deve incluir essa informação como JSON no corpo da solicitação. **Na maioria das vezes, você fornece as informações à `kubectl` em um arquivo .yaml**. `kubectl` converte a informação para JSON ao fazer a requisição para a API.
+
+Aqui está um exemplo de arquivo `.yaml` que mostra os campos necessários e as especificações de objeto para uma implatação Kubernetes:
+
+
+{{< codenew file="application/deployment.yaml" >}}
+
+Uma maneira de criar um Deployment usando um arquivo `.yaml` como o representado acima é usar o comando [`aplicar kubectl`](/docs/reference/generated/kubectl/kubectl-commands#apply
+) na interface de linha de comando `kubectl`, passando o arquivo `.yaml` como argumento. Aqui está um exemplo:
+
+```shell
+kubectl apply -f https://k8s.io/examples/application/deployment.yaml
+```
+
+A saída será similar a esta:
+
+```
+deployment.apps/nginx-deployment created
+```
+
+### Campos obrigatórios
+
+No arquivo `.yaml` para o objeto Kubernetes que pretende criar, você precisará definir valores para os seguintes campos:
+
+`apiVersion` - Qual a versão do API Kubernetes que está usando para criar este objeto.
+`kind` - Qual tipo de objeto pretende criar.
+`metadata` - Dados que ajudam a identificar de forma única o objeto, incluindo uma string `nome`, `UID` e um `namespace`.
+`spec` - Que estado deseja para o objeto.
+
+O formato preciso do objeto `spec` é diferente para cada objeto Kubernetes, e contém campos aninhados específicos para aquele objeto. O [Kubernetes API Reference](/docs/reference/kubernetes-api/
+) pode ajudá-lo a encontrar o formato de especificação para todos os objetos que você pode criar usando Kubernetes. 
+ 
+ 
+Por exemplo, veja o campo de `spec` field](/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec) para a referência Pod API. Para cada POD, o campo `.spec` especifica o pod e seu estado desejado (como o nome da imagem do contêiner para cada recipiente dentro daquela cápsula). Outro exemplo de especificação de um objeto é o [campo `spec` ](/docs/reference/kubernetes-api/workload-resources/stateful-set-v1/#StatefulSetSpec). Para o StatefulSet, o campo `.spec` especifica o StatefulSet e seu estado desejado. Dentro do `.spec` de um StatefulSet está um [template](/docs/concepts/workloads/pods/#pod-templates) para objetos de Pod. Esse modelo descreve os Pods que o controlador StatefulSet criará para satisfazer a especificação do StatefulSet. Diferentes tipos de objetos também podem ter diferentes `.status`; novamente, as páginas de referência API detalham a estrutura daquele campo `.status`, e seu conteúdo para cada tipo diferente de objeto.
```

Comment:
There is something wrong here: In the beginning of the statement, you are closing the "field" markdown but you never opened it.

Please break this line when it reaches the column 70 or 90 to make it easier to review, this statement is too big to be in a single line.

Also please check if [campo `spec` ] is rendered correctly on the preview, as this doesn't seems to work fine